src/java.base/share/classes/java/lang/StackStreamFactory.java

```diff
@@ -764,6 +763,10 @@ protected void initFrameBuffer() {
 
         @Override
         protected int batchSize(int lastBatchFrameCount) {
+            // this method is only called when the caller class is not found in
```

Comment:
Minor nit: start sentence with an uppercase letter.

Reply:
I'll leave it as is.